src/Timer.cpp

```diff
@@ -5,13 +5,22 @@
 
 #include "murmur_pch.h"
 
-// Ensure boost_system is header only.
-#define BOOST_ERROR_CODE_HEADER_ONLY
-// Ensure boost_chrono is header only.
-#define BOOST_CHRONO_DONT_PROVIDE_HYBRID_ERROR_HANDLING
-#define BOOST_CHRONO_HEADER_ONLY
+#include <boost/version.hpp>
 
-#include <boost/chrono.hpp>
+// Not all Boost versions can use a header-only
+// boost_system without build failures.
+//
+// See mumble-voip/mumble#2366.
+//
+// This was fixed in Boost 1.56.0.
+//
+// See:
+//  https://github.com/boostorg/system/blob/boost-1.56.0/include/boost/system/error_code.hpp#L514-L516
+// vs.
+//  https://github.com/boostorg/system/blob/boost-1.55.0/include/boost/system/error_code.hpp#L515-L517
+#if BOOST_VERSION >= 105900
```

Comment:
No, I had it to 1.59.0 first, because that was the version in the Windows build env I used.

This should be 105600